nimbusapp

```diff
@@ -57,6 +60,32 @@ function version_info() {
     output "Released on $NIMBUS_RELEASE_DATE"
 }
 
+function update_version() {
+    version_info
+
+    if prompt_yn "${C_BOLD}${C_RED}Do you want to update your nimbusapp version?${C_RESET}"; then
+        output "Updating nimbusapp version..."
+
+        if wget -nv -q "$NIMBUS_DOWNLOAD_URL" -O- | sudo tar -xzv -C "$NIMBUS_INSTALL_DIR" >/dev/null 2>&1; then
+            "${NIMBUS_INSTALL_DIR}/nimbusapp" --version
+        else
+            error "Updating nimbusapp version - version unchanged"
```

Comment:
We should check if they're up to date before asking to prompt them for an update. Otherwise it should just return "nimbusapp is up-to-date", akin to yum

Reply:
I think all we would accomplish is adding complexity. The only reliable way to check versions would be to download the latest and compare the output of the `version` commands. 

This is a convenience function, not a package manager. If we need to incorporate more versioning features I would like to use an existing system like npm, maven, pip or cpan.

Reply:
Fair point. Maybe we just differentiate errors between whether we're up to date or we failed to install (ie we don't have internet access).